spec/models/comment_spec.rb

```diff
@@ -25,4 +25,9 @@
     comment.text = nil
     expect(comment).to be_valid
   end
-end
+
+  it 'it should update post comment counter' do
+    comment.save
+    expect(post.comments_counter).to eq(0)
+  end
```

Comment:
- [ ] I suggest you test for a situation where a comment(s) is added as well, we would like to see the counter update.